### PR TITLE
add face to face docs

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: docs/walkthroughs/face-to-face.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Required docker images
+In order to reduce bandwidth consumption on the day, please pre-pull the required docker images before attending the Face 2 Face, using the following command:
+```
+for i in quay.io/3scale/apicast:master feedhenry/cordova-app-apb:0.0.6 feedhenry/ios-app-apb:0.0.6 feedhenry/android-app-apb:0.0.6 feedhenry/mcp-standalone:0.0.6 feedhenry/keycloak-apb:0.0.6 feedhenry/3scale-apb:0.0.6 feedhenry/aerogear-digger-apb:0.0.6 ansibleplaybookbundle/origin-ansible-service-broker:sprint139.1 openshift/origin-haproxy-router:v3.7.0-rc.0 openshift/origin-deployer:v3.7.0-rc.0 openshift/origin:v3.7.0-rc.0 openshift/origin-docker-registry:v3.7.0-rc.0 openshift/origin-pod:v3.7.0-rc.0 openshift/origin-service-catalog:v3.7.0-rc.0 quay.io/coreos/etcd:latest jimmidyson/keycloak-openshift:2.5.4.Final rhmap/redis:2.18.22; do docker pull $i; done
```
Comment (aidenkeating) [inline docs/walkthroughs/face-to-face.md]: We'll also need `aerogear/digger-android-slave-image:AGDIGGER-177` and `aerogear/digger-android-sdk-image:FH-v3.19` and `feedhenry/aerogear-digger-apb:0.0.6`

Comment (philbrookes) [reply docs/walkthroughs/face-to-face.md]: Cheers @aidenkeating `feedhenry/aerogear-digger-apb:0.0.6` is already there, I've added the other 2 now.

File: docs/walkthroughs/face-to-face.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Required docker images
+In order to reduce bandwidth consumption on the day, please pre-pull the required docker images before attending the Face 2 Face, using the following command:
+```
+for i in quay.io/3scale/apicast:master feedhenry/cordova-app-apb:0.0.6 feedhenry/ios-app-apb:0.0.6 feedhenry/android-app-apb:0.0.6 feedhenry/mcp-standalone:0.0.6 feedhenry/keycloak-apb:0.0.6 feedhenry/3scale-apb:0.0.6 feedhenry/aerogear-digger-apb:0.0.6 ansibleplaybookbundle/origin-ansible-service-broker:sprint139.1 openshift/origin-haproxy-router:v3.7.0-rc.0 openshift/origin-deployer:v3.7.0-rc.0 openshift/origin:v3.7.0-rc.0 openshift/origin-docker-registry:v3.7.0-rc.0 openshift/origin-pod:v3.7.0-rc.0 openshift/origin-service-catalog:v3.7.0-rc.0 quay.io/coreos/etcd:latest jimmidyson/keycloak-openshift:2.5.4.Final rhmap/redis:2.18.22 aerogear/digger-android-slave-image:AGDIGGER-177 aerogear/digger-android-sdk-image:FH-v3.19; do docker pull $i; done
+```
+
+## Local setup
+For the face to face, we are working from the 0.0.6 git tag.
+```
+git checkout 0.0.6
```
Comment (maleck13) [inline docs/walkthroughs/face-to-face.md]: maybe add the context of where to do the checkout?

**Comment by maleck13** [comment]
@aidenkeating can you try this out locally? I think a good test would be to remove any local images and then run the script in the doc, take a list of the images and hashes then run the installer, and provision the main services, there should be no new hashes or images

**Comment by philbrookes** [comment]
@aidenkeating let me know when you've finished trying this out, and I'll merge it then - assuming it works as expected.

**Comment by aidenkeating** [comment]
@philbrookes Had to update `feedhenry/fh-sync-server-apb:0.0.6` and create `feedhenry/fh-sync-server:0.0.6`. They also need to be added to the script. After that it looks to be working.

As discussed, was also seeing an error on installation:

```
TASK [ansible-service-broker-setup : Copy APB's from feedhenry docker org to feedhenry] ***
fatal: [localhost]: FAILED! => {"failed": true, "msg": "'dict object' has no attribute 'json'"}
	to retry, use: --limit @/Users/aidenkeating/dev/go/src/github.com/feedhenry/mcp-standalone/installer/playbook.retry

PLAY RECAP *********************************************************************
localhost                  : ok=18   changed=1    unreachable=0    failed=1
```

If we don't need to do this in the f2f, should we add a flag to skip it to the installation command.

Other than that looks good.

**Pull request merged**